README.md

```diff
@@ -0,0 +1,170 @@
+
+- **Show posts**
+- **Comment on posts**
+- **Like posts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Ruby (ruby --version)
+- SQLite3 (sqlite3 --version)
+- Node.js (node --version)
+- Yarn (yarn --version)
+
+### Install
+
+Since Rails started out as a project full of packages and libraries, and has since become the full-stack framework it is, it can be easy to forget that it is also just a package itself.
+If these are all good, you can check:
+
+- Rails (rails --version)
+
+If you get a good version, you're ready to go! (Almost).
+If you didn't, you may still need to install the Rails gem.
+
+`gem install rails`
+
+It can be good to keep in mind which version you're using as each of these will update on their own schedule and sometimes the code changes just enough to give us bugs. This will be true for all the packages, gems, APIs, or any other dependencies you may bring into your projects as a developer.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Amazinggracee/Blog-app.git
+```
+
+### Usage
+
+ITo begin with, you need to get your Rails application server running. This is like live preview in the browser for rails applications. Run :
+
+
+```sh
+ $ bin/rails server
+
+```
+
+OR
+
+```sh
+$ rails s
+
+```
+
+visit [localhost](http://localhost:3000/) in your browser!
+
+To exit server enter “Control (^ )+ C”
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ayesha Arshad**
+
+- GitHub: [@amazinggacee](https://github.com/Amazinggracee)
+- Twitter: [@amazinggaceu](https://twitter.com/amazinggraceu)
+- LinkedIn: [Amarachi Dimkpa](https://linkedin.com/in/amarachi-dimkpa)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Adding Authentication**
+- [ ] **Adding View elements**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give this repo a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Project architecture and requirments provided by Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[REQUIRED] Your license link currently takes me to a 404 page. Kindly fix this to make your project more professional.

Reply:
So, sorry, I have linked the right file. Thank you so much.